blockchain/blockchain.go

```diff
@@ -61,11 +61,12 @@ var (
 	blockProcessTimer  = metrics.NewRegisteredTimer("chain/process", nil)
 	blockFinalizeTimer = metrics.NewRegisteredTimer("chain/finalize", nil)
 	blockValidateTimer = metrics.NewRegisteredTimer("chain/validate", nil)
-	ErrNoGenesis       = errors.New("Genesis not found in chain")
+	ErrNoGenesis       = errors.New("genesis not found in chain")
 	ErrNotExistNode    = errors.New("the node does not exist in cached node")
 	ErrQuitBySignal    = errors.New("quit by signal")
 	ErrNotInWarmUp     = errors.New("not in warm up")
 	logger             = log.NewModuleLogger(log.Blockchain)
+	blockLogsPrefix    = []byte("blockLogs")
```

Comment:
How about specifying `kes` or something else? Later, it may confuse anyone without context.

Reply:
You're right. I changed it to `kesCachePrefixBlockLogs`.